.env.example

```
@@ -0,0 +1,40 @@
+FINAL_DATA_DIR="${PROCESSED}"
+
+# Where task parquet files live / are written.
+TASK_DIR="/path/to/eq_stuff/tasks"
+
+# Only needed for the `aces_to_eq` conversion pipeline. Leave unset otherwise.
+# ACES_SHARDS_DIR="/path/to/eic_stuff/make_index_dfs/task_configs"
+
+# Where training outputs and checkpoints land. Hydra creates
+# ${OUTPUT_DIR}/outputs/<date>/<time>/ run dirs underneath.
+SAVE_DIR="${PROJECT_DIR}/results/"
+OUTPUT_DIR="${PROJECT_DIR}/results"
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OUTPUT_DIR key should go before the SAVE_DIR key

(UnorderedKey)